websites/R/Reddit/presence.ts

```diff
@@ -1,39 +1,85 @@
-const presence: Presence = new Presence({
-    clientId: "609183409440555018"
-  }),
-  startTimestamp = Math.floor(Date.now() / 1000),
-  strings = presence.getStrings({
-    browsing: "presence.activity.browsing",
-    live: "presence.activity.live"
-  }),
-  oldReddit = document.querySelector("._1tvdSTbdxaK-BnUbzUIqIY") == null;
+let presence: Presence;
+
+function setPresence(): void {
+  if (location.pathname.includes("/r/netflix"))
+    presence = new Presence({ clientId: "869992823854870588" });
+  else presence = new Presence({ clientId: "609183409440555018" });
+}
+
+setPresence();
```

Comment:
Is this function even necessary xd

Reply:
Yea cause its needed to start updatedata and within updatedata if the pathname changes but the page doesnt get reloaded fully.

Reply:
No I mean there's no need to make it a function, and just use these statements, like isn't it the same

Reply:
![image](https://user-images.githubusercontent.com/34792838/128871993-585fbe86-c792-4808-b2fa-baf939ecfbec.png)
It's called twice
so instead of writing those 3 lines twice, just made it a small function